src/DocAssembler/README.md

```diff
@@ -0,0 +1,157 @@
+
+The second part is to get the numbers after the AB# text. This is configured here to be between 3 and 6 characters. We also want to reuse this ID in the value, so we capture it as a named subexpression called `id`.
+
+In the value we can reuse these named subexpression like this:
+
+```text
+${pre}[AB#${id}](https://dev.azure.com/[your organization]/_workitems/edit/${id})
+```
+
+We start with the `pre` value, after which we build a markdown link with AB# combined with the `id` as the text and the `id` as parameter for the URL. We reference an Azure Board work item here. Of course you need to replace the `[your organization]` with the proper value for your ADO environment here.
+
+### `Content`
+
+The content is defined with these properties:
+
+| Property         | Description                                                  |
+| ---------------- | ------------------------------------------------------------ |
+| `src` (Required) | The source sub-folder relative to the working folder.        |
+| `dest`           | An optional destination sub-folder path in the output folder. If this is not given, the relative path to the source folder is used. |
+| `files`          | This is a  [File Globbing in .NET](https://learn.microsoft.com/en-us/dotnet/core/extensions/file-globbing) pattern. Make sure to also include all needed files for documentation like images and assets. |
+| `exclude`        | This is a  [File Globbing in .NET](https://learn.microsoft.com/en-us/dotnet/core/extensions/file-globbing) pattern. This can be used to exclude specific folders or files from the content set. |
+| `rawCopy` | If this value is `true` then we don't look for any links in markdown files and therefor also don't fix them. This can be used for raw content you want to include in the documentation set like `.docfx.json`, templates and such. |
+| `urlReplacements`     | A collection of [`Replacement`](#replacement) objects to use for URL paths in this content set, overruling any global setting. These replacements are applied to calculated destination paths for files in the content sets. This can be used to modify the path. The generated template removes /docs/ from paths and replaces it by a /. More information can be found under [`Replacement`](#replacement). |
+| `contentReplacements` | A collection of [`Replacement`](#replacement) objects to use for content of files in this content set, overruling any global setting. These replacements are applied to all content of markdown files in the content sets. This can be used to modify for instance URLs or other content items. More information can be found under [`Replacement`](#replacement). |
+| `externalFilePrefix`  | The prefix to use for all referenced files in this content sets that are not part of the complete documentation set, like source files. It overrides any global prefix. This prefix is used in combination with the sub-path from the working folder. |
+
+
```

Comment:
Any example like you did before?
And also, I believe you have an extra line.

Reply:
Removed the extra line 😯 What kind of example would you like to see? This is just the detailed documentation of the classes. Example of the JSON is in line 58-90.

Reply:
Something like real life :-) Maybe from the tests even q bit simpler, just to give an idea

Reply:
I see. I think this would be more like a complete walkthrough - this is the folder structure, we want to change it in this. And I think it would be great to walk through one or more scenario's using the combination of tools to get to a result. I would like to modify the documentation part in the repo as a next step to include that. Also publish it to github.io pages to have all documentation in one location and more extended.

Would that be better?

Reply:
we can wait for another PR for a walk thru!